frontend/src/metabase/admin/settings/settings.js

```diff
@@ -71,6 +71,21 @@ export const sendTestEmail = createThunkAction(SEND_TEST_EMAIL, function() {
   };
 });
 
+export const CLEAR_EMAIL_SETTINGS =
+  "metabase/admin/settings/CLEAR_EMAIL_SETTINGS";
+export const clearEmailSettings = createThunkAction(
```

Comment:
Since it's not using the thunk functionality this can be simplified to the following since `clear()` returns a promise:

```
export const clearEmailSettings = createAction(
  CLEAR_EMAIL_SETTINGS,
  () => EmailApi.clear()
);
```